packages/app/src/vis-packs/core/utils.ts

```
@@ -71,7 +71,7 @@ export function applyMapping<T>(
 export function getSliceSelection(
   dimMapping?: DimensionMapping
 ): string | undefined {
-  if (!dimMapping || !dimMapping.some(isNumber)) {
+  if (!dimMapping?.some(isNumber)) {
```

The only linting fix unrelated to imports.

---

Not a fan of the combination of `!` with `?.`. I find it hard to process mentally.

---

Yeah agreed, I'll revert and silence the error